zerospend/src/main/java/io/coti/zerospend/services/DspVoteService.java

```diff
@@ -1,5 +1,6 @@
 package io.coti.zerospend.services;
 
+import afu.org.checkerframework.checker.oigj.qual.O;
```

Comment:
this is probably not needed and by mistake

Reply:
Yes, removed